### DISCUSSION TITLE
Creating a new transparent landscape

### DISCUSSION THREAD

**Discussion by Ddaniel84** [Stellarium/stellarium#2528] [General]
Hi, I succeed in creating a personnal landscape based upon "geneva" one. I'll improve it transforming the "opaque" part in a transparent one. I use this youtube movie as sample https://www.youtube.com/watch?v=xxHqRRYzdTM&t=917s. I create with Paint 3D a transparent.png file, an horizon file (the same as the previous and successfull one), and a . ini like that:
[landscape]
name = LMJOBStransp
author = Dominique DANIEL; 
description = Horizon line La Montjoie.
type = spherical
maptex = transparent.png
maptex_top = 40
maptex_bottom = -40
angle_rotatez=-90
tesselate_rowsz=40
tesselate_cols=40
polygonal_horizon_list = horizon_LMJOBStransp.txt
polygonal_angle_rotatez=0
polygonal_horizon_list_mode=azDeg_altDeg
horizon_line_color =  .0,.99,.99
minimal_brightness = 0.15
[location]
light_pollution = 7
atmospheric_extinction_coefficient = 0.29
atmospheric_temperature = 10
atmospheric_pressure = -1
planet = Earth
latitude = +44.096
longitude = +5.045
altitude = 110
timezone = Europe/Paris

Then I include these 3 files within a .zip one, and ask Stellarium to add the landscape.
Stellarium says me that .zip is not a landscape description.

Where is the issue? Thanks for your help.
Best 
Dominique (France)

**Comment by gzotti** [comment]
> Hi, I succeed in creating a personnal landscape based upon "geneva" one. I'll improve it transforming the "opaque" part in a transparent one. I use this youtube movie as sample https://www.youtube.com/watch?v=xxHqRRYzdTM&t=917s. I create with Paint 3D a transparent.png file, an horizon file (the same as the previous and successfull one), and a . ini like that: 
>[landscape] 
>name = LMJOBStransp 
>author = Dominique DANIEL; 
>description = Horizon line La Montjoie. 
>type = spherical 
>maptex = transparent.png 
>maptex_top = 40 
>maptex_bottom = -40 
>angle_rotatez=-90 
>tesselate_rowsz=40   <========= ERROR
>tesselate_cols=40 
>polygonal_horizon_list = horizon_LMJOBStransp.txt 
>polygonal_angle_rotatez=0 
>polygonal_horizon_list_mode=azDeg_altDeg 
>horizon_line_color = .0,.99,.99 
>minimal_brightness = 0.15 
>[location] 
>light_pollution = 7 
>atmospheric_extinction_coefficient = 0.29 
>atmospheric_temperature = 10 
>atmospheric_pressure = -1 
>planet = Earth 
>latitude = +44.096 
>longitude = +5.045 
>altitude = 110 
>timezone = Europe/Paris
> 
> Then I include these 3 files within a .zip one, and ask Stellarium to add the landscape. Stellarium says me that .zip is not a landscape description.
> 
> Where is the issue? Thanks for your help. Best Dominique (France)

The format is described in the Stellarium User Guide. Apart from the little typo, it seems to work. If not, attach the landscape ZIP or at least the logfile here.

**Comment by Ddaniel84** [comment]
Hi Thanks a lot for your kind proposal. Here is the zip file.
Dominique
[LMJOBStransp.zip](https://github.com/Stellarium/stellarium/files/9168652/LMJOBStransp.zip)

**Comment by Ddaniel84** [comment]
and this is the "opaque version" wich works..
[LMJ-OBS.zip](https://github.com/Stellarium/stellarium/files/9168664/LMJ-OBS.zip)

**Comment by gzotti** [comment]
The canonical filename is landscape.ini

**Comment by Ddaniel84** [comment]
Ohhh.. sorry .. But one renamed, the landscape is accepted.. but remains unvisible on the screen. May be colour definitions are wrong? there is no error message.

**Comment by gzotti** [comment]
If you have no real image, you can simply activate "Only polygon". However, here are the fixes.

[landscape]
; Definition file for including a horizon compatible with Carte du Ciel.
name = LMJOBStransp
; Do not copy now-wrong documentation lines from the template... 
author = Dominique DANIEL; 
description = Horizon line La Montjoie.  
type = spherical
maptex = transparent.png
maptex_top = 40
maptex_bottom = -40
; THE NEXT 3 ARE IRRELEVANT IN A TRANSP. TEXTURE
;angle_rotatez=-90
;tesselate_rowsz=40 <==== PLUS A TYPO HERE
;tesselate_cols=40
; Take care of capital letters in filenames if not on Windows!
polygonal_horizon_list = horizon_LMJOBStransp.txt
; THE NEXT LINE INDICATES THAT ZERO IS NOT A GOOD VALUE....
; in case the measured line was grid-relative only (meridian convergence issue) or had other problems, allow a corrective rotation also here.
; Unfortunately, a horizon with edges at exactly 0 or 180 degrees azimuth causes bad effects. Adding a minimal offset avoids this without causing much trouble.
; Because we can mix photo and polygon in the other classes, use a different name for the polygon rotation:
polygonal_angle_rotatez=0.00001 ; <<< THIS WORKS
polygonal_horizon_list_mode=azDeg_altDeg
; Color for the line. It will not be drawn if this is not defined. This color will not be dimmed at night.
horizon_line_color =  .0,.99,.99
; specify a minimum brightness value 0..1 to have the ground always visible.
minimal_brightness = 0.15
[location]
light_pollution = 7
atmospheric_extinction_coefficient = 0.29
atmospheric_temperature = 10
; The next line with value -1 translates to "compute from altitude and standard atmosphere model"
atmospheric_pressure = -1
planet = Earth
latitude = +44.096
longitude = +5.045
altitude = 110
timezone = Europe/Paris

**Reply by Ddaniel84** [reply]
Done.. that works! Plenty of thanks Georg.. You have been really kind. It's not as easy as I thought, but I have now a precious new information with Stellarium. I share the same with NINA, which has some instruction based on horizon visibility. 
And thanks for Stellarium that I love for several years! Dominique
